### ISSUE TITLE
Add metadata to project indicating contributor roles/skills

### ISSUE THREAD

**Issue by tilnow** [vgururao/yakcollective#74]
Jay Kalawar (US East Coast)Yesterday at 7:27 PM
@Maier (U+3) so one potential example that comes to mind is YC embeds a link on linkedin profile showing contribution to project, project mgmt, production etc. (like coursera etc do with their certs) with link to the finished product on YC. Would that work? Gives more credibility than adding two liner to one's own profile. Also helps build YC brand perhaps?
Maier (U+3)Yesterday at 7:52 PM
@Jay Kalawar (US East Coast) yes. maybe i would rephrase - yak puts a link to a project that allows them to ride on the respect that project gets. it also serves to shows a broader base of expertise.
i guess this means that part of the "project" format should be exactly this kind of information. i am not sure it does now. @Venkatesh Rao (US Pacific), what do you think about optimizing the project page format to boost yak cv-value?
Venkatesh Rao (US Pacific)Yesterday at 10:57 PM
I like it :smile:

**Comment by necopinus** [comment]
Probably makes most sense to add as a "credits" section at the end of the page. Maybe this should replace the current list in the header widget?
